src/pages/permissions/panels/CreateIdpGroupPanel.tsx

```diff
@@ -0,0 +1,232 @@
+  });
+
+  const rows = filteredGroups.map((group) => {
+    return {
+      name: group.name,
+      className: "u-row",
+      columns: [
+        {
+          content: group.name,
+          role: "cell",
+          "aria-label": "Name",
+        },
+        {
+          content: group.description || "",
+          role: "cell",
+          "aria-label": "Description",
+          title: group.description,
+        },
+      ],
+      sortData: {
+        name: group.name.toLowerCase(),
+        description: group.description.toLowerCase(),
+      },
+    };
+  });
+
+  const groupsTable = groups.length ? (
+    <ScrollableTable
+      dependencies={[groups, searchGroup, notify.notification, formik]}
+      tableId="idp-group-mapping-table"
+      belowIds={["panel-footer"]}
+    >
+      <SelectableMainTable
+        id="idp-group-mapping-table"
+        headers={headers}
+        rows={rows}
+        sortable
+        emptyStateMsg="No IDP groups found"
+        itemName="IDP group"
+        parentName=""
+        selectedNames={selectedGroups}
+        setSelectedNames={setSelectedGroups}
+        processingNames={[]}
+        filteredNames={groups.map((group) => group.name)}
+        hideContextualMenu
+      />
+    </ScrollableTable>
+  ) : (
+    <EmptyState
+      className="empty-state full-width show-left-icon"
+      image={<Icon name="plans" className="empty-state-icon" />}
+      title="No groups"
+    >
+      <p className="u-no-margin--bottom">There are no groups in the system.</p>
+      <Link to={`/ui/permissions/groups`}>
+        Create groups
+        <Icon className="external-link-icon" name="external-link" />
+      </Link>
+    </EmptyState>
+  );
+
+  return (
+    <>
+      <SidePanel isOverlay loading={isLoading} hasError={!groups}>
+        <SidePanel.Header>
+          <SidePanel.HeaderTitle>Create IDP group</SidePanel.HeaderTitle>
+        </SidePanel.Header>
+        <NotificationRow className="panel-notification" />
+        <SidePanel.Content className="u-no-padding">
+          <ScrollableContainer
+            dependencies={[notify.notification]}
+            belowIds={["panel-footer"]}
+          >
+            <Form onSubmit={formik.handleSubmit}>
+              {/* hidden submit to enable enter key in inputs */}
+              <Input type="submit" hidden value="Hidden input" />
+              <Input
+                type="text"
+                label="Name"
+                required
+                autoFocus
+                id="name"
+                name="name"
+                onBlur={formik.handleBlur}
+                onChange={formik.handleChange}
+                value={formik.values.name}
+                error={formik.touched.name ? formik.errors.name : null}
+                placeholder="Enter name"
+              />
+            </Form>
+            <PermissionGroupsFilter
+              onChange={setSearchGroup}
+              value={searchGroup}
+            />
+            {groupsTable}
```

Comment:
I think the section to search and select groups needs a heading to clarify what it is about. Something like "Select groups to be mapped to the IDP group you are creating", but shorter and more precise.

![image](https://github.com/canonical/lxd-ui/assets/1155472/621c2a41-49db-4e84-af69-a2e285fb089b)